app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
+
+    def repr(self) -> str:
+        return f"Person(name={self.name}, age={self.age})"
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    Person.people = {}
```

Comment:
this line redundant here

Reply:
Person.people = {} is redundant because you have this code in 2 line